charts/harbor-operator/templates/crds.yaml

```diff
@@ -4116,6 +4116,28 @@ spec:
                         - authenticationURL
                         - container
                         type: object
+                      oss:
+                        properties:
+                          accessKeyID:
+                            type: string
+                          accessSecretRef:
+                            type: string
+                          bucket:
+                            type: string
+                          endpoint:
+                            type: string
+                          pathPrefix:
+                            type: string
+                          serverSideEncryption:
```

Comment:
Why is it defined inside chartmuseum CRD while it seems that chartmuseum doesn't use it